examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
```

Comment:
I'd argue it is not necessary to have redis pool because it does not really do anything.